src/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
-import ReactDOM from 'react-dom/client'
-import { createGlobalStyle, ThemeProvider } from 'styled-components'
+import reactDom from 'react-dom/client'
+import { ThemeProvider, createGlobalStyle } from 'styled-components'
```

Comment:
This is the way I handle things. If I add a TODO it's because I want the PR to focus on a goal for which the TODO task in mot a priority. It's just a reminder for later on so that I don't forget to handle these tasks.

No need to comment my TODOs.